quiz.js

```diff
@@ -20,9 +22,26 @@ var treeStuff ={};
 };
 
 submit.addEventListener("click" , tree)
-console.log(space)
-//have to set up listener before vaules declared so that they don't get returned null
-//adding this line to do another PR
+
+char.addEventListener("keyup", function(event) {
+	var newRows = enter.value;
+	var newChar = char.value;
+	event.preventDefault();
+    if (newRows.length > 1 || newChar.length > 1) {
+    	alert("ONLY ONE CHARACTER");
+    	    }
+     else if (event.keyCode == 13) {
+        document.getElementById("submit").click();
+    }
+
+    else if(event.keyCode < 48 || event.keyCode > 57 || event.keyCode != 8){
+        alert ("Only CHARACTERS no Letters");
+    }
```

Comment:
I like that you're trying something different for form validation, but it's not working as you expect. If I use `tab` to jump from input to input it pops up the alert error. Also, hitting `shift` to type any character on the number keys does the same thing. It also won't work for hitting `enter` to submit the form inputs. It even threw an error when I used `command + tab` to jump from one browser to another when I had your app running : )